openshift-kube-apiserver/filters/apirequestcount_filter.go

```diff
@@ -16,6 +16,14 @@ func WithAPIRequestCountLogging(handler http.Handler, requestLogger apirequestco
 		if !ok || !info.IsResourceRequest {
 			return
 		}
+		gvr := schema.GroupVersionResource{
+			Group:    info.APIGroup,
+			Version:  info.APIVersion,
+			Resource: info.Resource,
+		}
+		if _, ok = apirequestcount.DeprecatedAPIRemovedRelease[gvr]; !ok {
```

Comment:
need to check if the current version is past the version we removed the API in.  Once we're no longer serving, the user is effectively warned/broken already and can notice on their own.

Reply:
d6dfede072 -> 15019c4 turned this line into:

```golang
if minor, ok := apirequestcount.DeprecatedAPIRemovedRelease[gvr]; !ok || minor < currentMinor
```

So this thread can probably be marked resolved, although I don't have the power to do that myself.